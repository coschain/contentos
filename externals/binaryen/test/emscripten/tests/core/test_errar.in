#include <stdio.h>
#include <errno.h>
#include <string.h>

int main() {
  char* err;
  char buffer[200];

  err = strerror(EDOM);
  strerror_r(EWOULDBLOCK, buffer, 200);
  printf("<%s>\n", err);
  printf("<%s>\n", buffer);

  printf("<%d>\n", strerror_r(EWOULDBLOCK, buffer, 0));
  errno = 123;
  printf("<%d>\n", errno);

  return 0;
}
