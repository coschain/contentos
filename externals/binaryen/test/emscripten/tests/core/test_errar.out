<Domain error>
<Resource temporarily unavailable>
<34>
<123>
